src/ApiMethodDocumentation.js

```diff
@@ -826,7 +827,7 @@ export class ApiMethodDocumentation extends AmfHelperMixin(LitElement) {
       </div>`}
     </div>
     ${methodSummary ? html`<p class="summary">${methodSummary}</p>` : ''}
-    ${operationId ? html`<span class="operation-id">Operation ID: ${operationId}</span>` : ''}
+    ${operationId && !isAsyncApi ? html`<span class="operation-id">Operation ID: ${operationId}</span>` : ''}
```

Comment:
Isn't the field supposed to have been added for asyncApi ? If that is the case `operationId && !isAsyncApi` is confusing 🤔

Reply:
The field was created, I added this field in api URL component for async api so when it is async api shouldn't show in this part and shows in api url